build/test_encrypt.py

```diff
@@ -1,47 +1,42 @@
 """This is unit test for text file Encryption 
 and Decryption.
 """
-    
-import unittest
-import os
 from binascii import Error
 import cryptography
 from cryptography.fernet import Fernet
+import unittest
 
-class TestEncryptDecrypt(unittest.TestCase):
+class TestEncryption(unittest.TestCase):
     def setUp(self):
-        self.key_path = "D:\zip file version\client-server-network-combatzone-2\static\crypto.key"
-        self.file_path = "D:\zip file version\client-server-network-combatzone-2\static\serdes.dat"
-
-        # Generate and save a key
-        key = Fernet.generate_key()
-        with open(self.key_path, "wb") as crypto_file:
-            crypto_file.write(key)
-
-    def test_encrypt_decrypt(self):
+        # Generate and save the key
+        write_key()
         # Load the key
-        key = open(self.key_path, "rb").read()
-
-        # Encrypt the file
-        f = Fernet(key)
-        with open(self.file_path, "rb") as file:
-            data = file.read()
-        encrypted_data = f.encrypt(data)
-        with open(self.file_path, "wb") as encrypted_file:
-            encrypted_file.write(encrypted_data)
-
-        # Decrypt the file
-        f = Fernet(key)
-        with open(self.file_path, "rb") as encrypt_file:
-            encrypted_data = encrypt_file.read()
-        decrypted_data = f.decrypt(encrypted_data)
-
-        # Check that the decrypted data is the same as the original data
-        with open(self.file_path, "rb") as file:
-            original_data = file.read()
+        self.key = load_key()
+
+    def test_encrypt(self):
+        # Test that the encrypt function returns the expected output
+        encrypt("D:\my project code v1\client-server-network-master\static\crypto.key.key", self.key)
+        with open("D:\my project code v1\client-server-network-master\static\crypto.key.key", "rb") as encrypted_file:
+            encrypted_data = encrypted_file.read()
+        # Check that the encrypted data is different from the original data
+        with open("test.txt", "rb") as original_file:
+            original_data = original_file.read()
+        self.assertNotEqual(encrypted_data, original_data)
+
+    def test_decrypt(self):
+        # Test that the decrypt function returns the expected output
+        encrypt("test.txt", self.key)
+        decrypt("test.txt", self.key)
+        with open("test.txt", "rb") as decrypted_file:
+            decrypted_data = decrypted_file.read()
+        # Check that the decrypted data is equal to the original data
+        with open("test.txt", "rb") as original_file:
+            original_data = original_file.read()
         self.assertEqual(decrypted_data, original_data)
+file = "D:\my project code v1\client-server-network-master\static\crypto.key.key"
 
-   
-
-if __name__ == "__main__":
+if __name__ == '__main__':
```

Comment:
```suggestion
if __name__ == '__main__':
      **unittest.main()
      test_encrypt(self)
      test_decrypt(self)**
```
this duplicates here, we should keep only one and add all other function calls under that one line like the suggested example.